gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -207,6 +210,124 @@ def __repr__(self):
         [str(s) for s in [self.name, self.genotype, self.phaseset, self.info]])
 
 
+class _ToVcfRecordCoder(coders.Coder):
+  """Coder for encoding :class:`Variant` objects as VCF text lines."""
+
+  def encode(self, variant):
+    """Converts a :class:`Variant` object back to a VCF line."""
+    encoded_info = self._encode_variant_info(variant)
+    format_keys = self._get_variant_format_keys(variant)
+    encoded_calls = self._encode_variant_calls(variant, format_keys)
+
+    columns = [
+        variant.reference_name,
+        variant.start if variant.start is None else variant.start + 1,
```

Comment:
nit: I think it's better to rephrase as "variant.start + 1 if variant.start is not None else None" (feels strange to assign it to variant.start if it's None).

Reply:
Done.